app/models/relationship.rb

```diff
@@ -167,10 +167,17 @@ def self.create_unless_exists(source_id, target_id, relationship_type, options =
           r.send("#{key}=", value) if r.respond_to?("#{key}=")
         end
         r.save!
-      rescue
+      rescue StandardError => e
+        logger.error("[Relationship::create_unless_exists] rescuing #{e}")
```

Comment:
Opinion: This log entry, as is, is not super useful since the `e` object, when printed, won't contain all the information. I suggest we also include the `source_id` and `target_id`, or just drop this line, since it's going to be reported to Sentry anyway.

Minor: in models, we usually set `Rails.logger` instead of just `logger`, so I would rather keep it only for consistency.

Reply:
Done. I've removed the log entry here. I agree it's not terribly useful. I guess I mostly wanted to understand what are the reasons for this rescue block to be invoked (rather than the specifics). Given we'll have details in Sentry, I agree on removing the logging line and have done so.